packages/lib/gpcircuits/circuits/numeric-value.circom

```diff
@@ -32,7 +32,17 @@ template NumericValueModule() {
     signal input minValue;
     signal input maxValue;
 
+    // Absolute value of minimum value of a 64-bit signed integer.
+    // This will be added to all values fed into the bounds check
+    // module to convert them to 64-bit unsigned integers while
+    // preserving order.
+    var ABS_POD_INT_MIN = 1 << 63;
```

Comment:
I love how simple this is, now that we've thought through it.  Hopefully it stays that way when we add other types of comparison and arithmetic.